app/models/compute_resources/foreman/model/libvirt.rb

```diff
@@ -148,6 +148,9 @@ def new_vm(attr = { })
       opts[:boot_order] = %w[hd]
       opts[:boot_order].unshift 'network' unless attr[:image_id]
 
+      # Force CPU host-passthrough
+      opts[:cpu] ||= {}
```

Comment:
Note to self: this doesn't work because it matches fog-libvirt's default. fog-libvirt checks `.empty?` and that's still the same